src/rebar_hex_repos.erl

```diff
@@ -8,7 +8,7 @@
 
 -ifdef(TEST).
 %% exported for test purposes
--export([repos/1, merge_repos/1]).
+-export([repos/1, merge_repos/1, auth_config_file/1]).
```

Comment:
you may want to unexport this since it's unused in tests

Reply:
Yup, left over, good catch 👍